static/docs/commands-reference/version.md

```diff
@@ -11,6 +11,9 @@ usage: dvc version [-h] [-q | -v]
 
 ## Description
 
+> Like `dvc init`, this is one of the few commands that doesn't require an
```

Comment:
I'm not sure this is really important )

Reply:
You're right. I guess it's obvious. Deleted in a7ee6aa